.travis.yml

```diff
@@ -6,15 +6,19 @@ cache:
     directories:
         - "node_modules"
 env:
-    - FX_VERSION="54.0b"
-    - FX_VERSION="45.0.2esr"
+    global:
+        secure: ""
+    matrix:
+        - FX_VERSION="54.0b"
+        - FX_VERSION="45.0.2esr"
 matrix:
     fast_finish: true
     #allow_failures:
     #    - env: FX_CHANNEL="beta"
 notifications:
     email: false
 install:
+    - gem install dpl
```

Comment:
We should make this conditional too so that it doesn't run for the ESR, right? We should either just do this in the same line below or, if we want to do this in the install stage, define an environmental variable with all the conditions (FX_VERSION, TRAVIS_REPO_SLUG, etc.) and use that here and below.